Wikipedia/Code/ArticleCollectionViewCell+WMFFeedContentDisplaying.swift

```diff
@@ -84,3 +84,28 @@ public extension ArticleCollectionViewCell {
         setNeedsLayout()
     }
 }
+
+public extension RankedArticleCollectionViewCell {
+    override func configure(article: WMFArticle, displayType: WMFFeedDisplayType, index: Int, count: Int, layoutOnly: Bool) {
+        rankView.rank = index + 1
+        let startColor = UIColor.wmf_blue
+        let endColor = UIColor.wmf_green
+        var r1: CGFloat = 0
+        var g1: CGFloat = 0
+        var b1: CGFloat = 0
+        var a1: CGFloat = 0
+        startColor.getRed(&r1, green: &g1, blue: &b1, alpha: &a1)
+        var r2: CGFloat = 0
+        var g2: CGFloat = 0
+        var b2: CGFloat = 0
+        var a2: CGFloat = 0
+        endColor.getRed(&r2, green: &g2, blue: &b2, alpha: &a2)
+        let percent = CGFloat(index + 1) / CGFloat(count)
+        let r = r1 + percent * (r2 - r1)
+        let g = g1 + percent * (g2 - g1)
+        let b = b1 + percent * (b2 - b1)
+        let a = a1 + percent * (a2 - a1)
```

Comment:
Alpha is expected to always be one in this case. I wrote the code to be generalizable (because it could be re-usable elsewhere where you might want a gradient with an alpha change).